renderer/index.js

```diff
@@ -592,59 +593,73 @@ function addTorrent (torrentId) {
   ipcRenderer.send('wt-start-torrenting', torrentKey, torrentId, path)
 }
 
-function addSubtitle (file, autoSelect) {
-  var srtToVtt = require('srt-to-vtt')
-  var LanguageDetect = require('languagedetect')
-
+function addSubtitles (files, autoSelect) {
   // Subtitles are only supported while playing video
   if (state.playing.type !== 'video') return
 
-  // No dupes allowed
+  // Read the files concurrently, then add all resulting subtitle tracks
+  console.log(files)
   var subs = state.playing.subtitles
-  var filePath = file.path || file
-  if (subs.tracks.some((track) => track.filePath === filePath)) return
+  Async.map(files, loadSubtitle, function (err, tracks) {
+    if (err) return onError(err)
+
+    for (var i = 0; i < tracks.length; i++) {
+      // No dupes allowed
+      var track = tracks[i]
+      if (subs.tracks.some((t) => track.filePath === t.filePath)) continue
```

Comment:
In general, I prefer not abbreviating object variables that we're later going to modify since it makes it harder to see what's being modified. I think it's fine for other types of variables, or when we're just reading the value.

It's longer, but clearer IMO:

```js
state.playing.subtitles.tracks.push(track)
```